golos.publication/golos.publication.cpp

```diff
@@ -317,6 +317,11 @@ void publication::payto(name user, eosio::asset quantity, enum_t mode, std::stri
     if(quantity.amount == 0)
         return;
 
+    if (!golos::vesting::balance_exist(config::vesting_name, user, quantity.symbol.code())) {
```

Comment:
`payto` function it's good place for this check. But `payto` pays reward in two mode: token and vesting. In this cases we need different checks. And in additional it's be better to add payment information in memo.